web-app/frontend/src/instruments/helpers/AddNameManager.ts

```diff
@@ -65,6 +65,25 @@ export class AddNameManager {
       this.updateRemoveButtons(); // Ensure correct behavior when rows are removed
     });
 
+    // Add event listener for left/right direction and alignment of the name input based on language selection
+    const langInput = row.querySelector(
+      `#language${index}`,
+    ) as HTMLInputElement;
+    const nameInput = row.querySelector(`#name${index}`) as HTMLInputElement;
+
+    langInput.addEventListener('change', () => {
+      const lang = this.languages.find(
+        (l) => l.wikidata_code === langInput.value,
+      );
+
+      if (lang) {
+        nameInput.classList.add(
+          lang.html_direction === 'rtl' ? 'force-rtl' : 'force-ltr',
+        );
+        nameInput.style.textAlign =
+          lang.html_direction === 'rtl' ? 'right' : 'left';
```

Comment:
Is this still needed?

Reply:
I think it's better to make `.force-rtl` and `.force-ltr` than doing this. Just had a closer look at them, you can try move them out of the nest they are in right now.